src/backend/input.rs

```diff
@@ -494,7 +494,9 @@ impl Pointer {
                         .matrix3
                         .mul_scalar(1.0 - 0.025 * self.now.scroll);
                 } else {
-                    grab_data.offset.z -= self.now.scroll * 0.05;
+                    if overlay.state.slideable {
```

Comment:
i don't think we want to introduce an `OverlayState.slideable` for this

let's pass `state: &AppState` to `handle_grabbed` and then read `state.session.config.screens_slideable` here. that way, we can make a toggle button for that in the settings panel, instead of it being read once at startup

Reply:
Alright

I ended up passing just `config: &GeneralConfig` because the borrow checker was not happy about passing the whole Appstate, as the pointer `handle_grabbed` is operating on is also a reference from `app: &mut Appstate`